samples/parallax-scrolling-aseprite/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.soywiz.korge.parallaxscrolling.aseprite">
```

Comment:
Was this file needed on samples?

Reply:
I just copy-pasted it. Don't remember from where actually ;)  I can be removed I guess.

Reply:
File removed